.ahub/tcchecker-tca/config.yaml

```diff
@@ -122,16 +124,20 @@ test:
       - /compiler/luci-eval-driver
       - /compiler/luci-pass-value-test
       - /compiler/luci-value-test
-      - /compiler/mio-circle
+      - /compiler/mio-circle04
       - /compiler/mio-tflite
+      - /compiler/mio-tflite260
       - /compiler/oops
       - /compiler/pepper-assert
+      - /compiler/pepper-csv2vec
       - /compiler/pepper-str
       - /compiler/pepper-strcast
       - /compiler/pp
+      - /compiler/rawdata2hdf5
       - /compiler/record-minmax
       - /compiler/safemain
       - /compiler/souschef
+      - /compiler/tf2tfliteV2
```

Comment:
Q) this is python script module that is used in import of TF models. is this needed for TCM?

Reply:
Maybe not. I added it based on preset list, and didn't check implementation.
I'll remove this.